File: be/src/main/java/com/project/popupmarket/dto/payment/ReceiptsTO.java
package com.project.popupmarket.dto.payment;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Builder;
import lombok.Getter;

import java.math.BigDecimal;
import java.time.LocalDate;

@Getter
@Builder
@JsonIgnoreProperties(ignoreUnknown = true)
public class ReceiptsTO {
    private String paymentKey;
    private String orderId;
    private Long customerId;
    private Long landId;
    private LocalDate start;
    private LocalDate end;
    private BigDecimal amount;

    public ReceiptsTO withCustomerId(Long customerId) {
        this.customerId = customerId;
        return this;
    }
}

File: be/src/main/java/com/project/popupmarket/dto/payment/ReceiptsTO.kt
package com.project.popupmarket.dto.payment

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.math.BigDecimal
import java.time.LocalDate

@JsonIgnoreProperties(ignoreUnknown = true)
data class ReceiptsTO(
    val paymentKey: String,
    val orderId: String,
    var customerId: Long,
    val landId: Long,
    val start: LocalDate,
    val end: LocalDate,
    val amount: BigDecimal
)
